starter_code/src/movies.js

```diff
@@ -1,20 +1,79 @@
-/* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
+function turnHoursToMinutes(arr) {
+  var finalArray = arr.map(function(movie) {
+    var newMovie = movie;
+    if (movie.duration.length === 1 && movie.duration.includes('min')) {
+      newMovie.duration = parseInt(movie.duration.replace('min', '')) * 60;
+      return newMovie;
+    } else if (movie.duration.length === 1 && movie.duration.includes('h')) {
+      newMovie.duration = parseInt(movie.duration.replace('h', ''));
+      return newMovie;
+    } else {
+      newMovie.duration =
+        parseInt(movie.duration.split('h')[0]) +
+        parseInt(movie.duration.split('h')[1].replace('min', ''));
```

Comment:
Me gusta esa "vaina" --> Replace!!! Si aprendéis expresiones regulares, replace os convertira en ninjas